src/app/component/user/components/profile/eco-places/eco-places.component.ts

```diff
@@ -1,22 +1,30 @@
 import { Component, OnInit } from '@angular/core';
+import { EcoPlaces } from '@user-models/ecoPlaces.model';
+import { ProfileService } from '@global-user/components/profile/profile-service/profile.service';
 
 @Component({
   selector: 'app-eco-places',
   templateUrl: './eco-places.component.html',
   styleUrls: ['./eco-places.component.scss']
 })
 export class EcoPlacesComponent implements OnInit {
-  private ecoPlaces: Array<string> = [
-    'Everyday Bakery Cafe', 'Culturist', '3 бобра',
-    'Everyday Bakery Cafe', 'Culturist', '3 бобра',
-    'Everyday Bakery Cafe', 'Culturist', '3 бобра',
-    'Everyday Bakery Cafe'];
+  public ecoPlaces: EcoPlaces[] = [];
+  public error = null;
 
-  constructor() {}
+  constructor(private profileService: ProfileService) { }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.getEcoPlaces();
+  }
 
-  public getEcoPlaces(): Array<string> {
-    return this.ecoPlaces.slice(0, 3);
+  public getEcoPlaces(): EcoPlaces[] {
+    return this.profileService.getEcoPlaces().subscribe(
+      (success) => {
```

Comment:
If you have only one assignment or one fuction call, you don't need to write it in separate lines, so please, rewrite next three lines in one line.

Reply:
done